.github/CONTRIBUTING.md

```diff
@@ -39,7 +39,7 @@ There are a few basic ground-rules for contributors (including the maintainer(s)
 
```

Comment:
Replace `Parity Ethereum Style Guide`?

Reply:
I've it out of scope of this PR, because it points to wiki website that we won't change. Open Ethereum doesn't have a website for now, so all the wiki links will need to be changed sometime later.

Reply:
Ok, I see. After this PR merged we could maybe add an issue for it because it's easily forgotten. Also, external contributors seem to like fixing documentation :)

Reply:
Yep, definitely. That's why this PR is called "initial cleanup", I'm not fixing everything here :)